sale_block_no_stock/models/res_company.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    sale_line_field_block = fields.Many2one(
+        comodel_name="ir.model.fields",
+        string="Field to compare quantities on Sale Order Lines",
```

Comment:
suggestion: I think specifying the type of quantity or check will help understand how this works.

```suggestion
        string="Field to compare available quantities on Sale Order Lines",
```

Reply:
Depends on the field you choose. Available means an specific field on the line. This way leaves the dependency of the meaning with another field.